plugins/CMakeLists.txt

```diff
@@ -12,12 +12,16 @@ if(BRAYNS_NETWORKING_ENABLED)
   add_subdirectory(Rockets)
 endif()
 
-option(BRAYNS_VRPN_ENABLED "Activate VRPN plugin" OFF)
+if(BRAYNS_OPENDECK_ENABLED)
```

Comment:
The option should go here then, not in the top-level CMake?!

Reply:
Done